### PR TITLE
Develop.voucher send by mail changes

### PR DESCRIPTION
**Pull request by GerbenBosschieter**

### CODE REVIEW

**Comment by GerbenBosschieter** [comment]
https://github.com/teamforus/forus/issues/1401

File: resources/lang/nl/mails/implementations/general/voucher_sent.php [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
+    'title' => 'Hierbij ontvangt u uw :fund_name-voucher',
     'you_have_asked' => 'U heeft gevraagd uw :product_name-voucher per e-mail te ontvangen.',
+    'voucher_details' => 'De voucher heeft een waarde van € :voucher_amount en is geldig tot en met :expire_at_minus_day.'.
+        'Onderstaande voucher laat u bij de organisatie van uw keuze zien. ',
+    'automatic_payment' => 'De organisatie scant deze code en de betaling gebeurt automatisch.',
     'qr_code_under' => 'Onderstaande QR-code laat u bij de aanbieder zien.',
-    'provider_scans' => 'De aanbieder scant deze code en de betaling gebeurt automatisch. Verder hoeft u geen actie te ondernemen..'
-];
+    'provider_scans' => 'De aanbieder scant deze code en de betaling gebeurt automatisch.',
+    'purchase_notice'   => 'Let op! Als u een aanbieding afneemt met een geldbedrag lager dan € :voucher_amount, blijft het restantbedrag op de voucher staan.  '.
```
Comment (maxvisser) [inline resources/lang/nl/mails/implementations/general/voucher_sent.php]: i would remove the notice. As it is a bit weird in context of a product voucher

**Comment by GerbenBosschieter** [comment]
@maxvisser pleas review again

**Comment by maxvisser** [comment]
you are not setting the new variables in calling the function of the service https://github.com/teamforus/forus-backend/blob/61acb3398d39241187e90c8a80aaf80572f01ff1/app/Models/Voucher.php#L206

**Comment by GerbenBosschieter** [comment]
@maxvisser and now ?

**Comment by maxvisser** [comment]
Werkt !

**Pull request merged**